.github/bin/s3/setup-empty-s3-bucket.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+S3_SCRIPTS_DIR="${BASH_SOURCE%/*}"
+
+S3_BUCKET_IDENTIFIER=trino-s3fs-ci-$(openssl rand -hex 8)
+
+S3_BUCKET_TTL=$(date -u -d "+2 hours" +"%Y-%m-%dT%H:%M:%SZ" 2>/dev/null || date -u -v "+2H" +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
why `||`? document

Reply:
It looks like there are different methods for adjusting the date/time based on the version of the command? I took this line from the `setup-aws-redshift.sh` script. On my local machine it uses `-v` to specify the offset, but I'm guessing that on other machines it uses `-d`. They both do the same thing, they just use a different format.

Reply:
So if the version of the date function doesn't support `-d`, then it will use `-v`

Reply:
Looks like the difference is between linux and osx

Reply:
Document please